### ISSUE TITLE
Quote icon is missing alt text

### ISSUE THREAD

**Issue by srimanh** [kalviumcommunity/devlinks-03#18]
The icon <img> in src/components/Header.jsx has no alt attribute. Add descriptive alt text.
